package.json

```diff
@@ -55,7 +55,7 @@
     "@guardian/src-text-area": "^2.8.1",
     "@guardian/src-text-input": "^2.7.1",
     "@guardian/types": "github:guardian/types#semver:1.1.0",
-    "@guardian/renditions": "git+https://github.com/guardian/renditions.git#0.1.0",
+    "@guardian/renditions": "git+https://github.com/guardian/renditions.git#0.1.1",
```

Comment:
Why the git URL (`git+`) rather than the GitHub one (`github:`) out of curiosity?

Reply:
I think it's usually added before the protocol, unless the protocol is `git://`. 
`npm init` formats it this way when you init with a git repo.

Reply:
https://docs.npmjs.com/cli/v6/configuring-npm/package-json#git-urls-as-dependencies

Reply:
Is there a reason to prefer that over the `github:` URL? The next section in that docs page ^: https://docs.npmjs.com/cli/v6/configuring-npm/package-json#github-urls

Reply:
Lol didn't know about this and that was just the next paragraph down 🙈
Looks like the github URL usage is as simple as `user/repo#...` to which I might only ever so slightly prefer having an explicit github url so it doesn't confuse people where this is being pulled from (i.e. looking for it in the npm registry). 
Do you have a preference @JamieB-gu ?